app-misc/neofetch/neofetch-3.3.0.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A CLI system information tool written in BASH that supports displaying images"
+DESCRIPTION="Neofetch displays information about your system next to an image, your OS logo, or any ASCII file of your choice. The main purpose of Neofetch is to be used in screenshots to show other users what OS/Distro you're running, what Theme/Icons you're using etc"
+HOMEPAGE="https://github.com/dylanaraps/neofetch"
+COPYRIGHT="2016-2017 Dylan Araps"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dylanaraps/$portName/archive/$portVersion/$portVersionedName.tar.gz"
+CHECKSUM_SHA256="4808e76bd81da3602cb5be7e01dfed8223b1109e2792755dd0d54126014ee696"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	neofetch
+	cmd:neofetch
+	"
+REQUIRES="
+	haiku
+	cmd:bash
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+GLOBAL_WRITABLE_FILES="settings/neofetch/config.conf"
```

Comment:
What do you think of the following suggestion ?
```sh
BUILD_PREREQUIRES="
	cmd:make
	"
```
then, after the GLOBAL_WRITABLE_FILES stuff,
```
PATCH()
{
	sed -i \
		-e "s|/share/neofetch/|/$relativeDataDir/neofetch/|;" \
		Makefile
}

INSTALL()
{
	make install PREFIX=$prefix MANDIR=$manDir SYSCONFIGDIR=$sysconfDir
}
```

It's not finished yet because it does not copy the template file:
data/neofetch/config.conf

But with 1 or 2 extra commands in INSTALL we should obtain the same result as with the current INSTALL.

Reply:
Do $sysconfDir exist ?
I don't see it on haikuports wiki

Reply:
Exact. $sysconfDir is not in the wiki. But it exists:
```sh
haikuports -S neofetch
cd /path/to/haikuports/app-misc/neofetch/work-3.3.0
grep ^sysconfDir wrapper-script
sysconfDir='/packages/neofetch-3.3.0-1/.settings'
```
And it is being used in more than 30 recipes, so I guess it is OK to use it in this one too ;-)

(the -S in not necessary. It's just that I like to use that switch.)

Reply:
Changed. Thanks